MavenQuickStart/quickApp/src/main/java/com/globant/topicthree/exerciseone/UserOperationsController.java

```diff
@@ -0,0 +1,57 @@
+package com.globant.topicthree.exerciseone;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * API implementation of UserService for virtual Shopping Cart (CRUD).
+ * 
+ * @author andres.vaninetti
+ *
+ */
+public class UserOperationsController implements UserService {
+
+	private List<User> userDAOList;
+
+	public UserOperationsController() {
+		this.userDAOList = new ArrayList<User>();
+	}
+
+	public void createUser(int id, String name) {
+		User userToCreate = new User(id, name);
+		this.userDAOList.add(userToCreate);
+	}
+
+	public User readUser(int id) {
+		if (this.userDAOList != null) {
+			for (User user : this.userDAOList) {
```

Comment:
If you used a map with the key being the ID, you wouldn't need to traverse the entire array. Is more performant that way